test/addapibackend.js

```diff
@@ -32,76 +37,107 @@ test.describe('Add API Backend', function() {
         CommonUtils.signIn(driver);
-        CommonUtils.fillSignInForm(driver, 'test@test.test', 'testuser');
+        CommonUtils.fillSignInForm(driver, 'robot@test.com', 'robottest');
         CommonUtils.goToDashboard(driver);
         var values = {
-            backendName: 'NewBackend',
-            description: 'Test Description',
-            hostName: '',
-            portNumber: '80',
+            apiName: 'NewBackend',
+            apiDescription: 'Test Description'
         };
         AddAPIBackendUtil.clickAddNewBackend(driver);
-        AddAPIBackendUtil.fillBaseInformation(driver, values);
-        AddAPIBackendUtil.fillBackendInformation(driver, values);
-        var hostRequiredErrorElement = driver.findElement(By.xpath('//*[@id="backend-information-form"]/div[2]/span'));
+        AddAPIBackendUtil.fillApiName(driver, values);
+        AddAPIBackendUtil.fillApiDescription(driver, values);
+        AddAPIBackendUtil.savingInformation(driver);
+        // Get the URL field in form
+        var hostRequiredErrorElement = driver.findElement(By.xpath('//*[@id="addApiForm"]/fieldset/div[3]/span'));
         hostRequiredErrorElement.getText().then(function(text) {
-            assert.equal(text, 'Host is required');
+            assert.equal(text, 'URL is required');
         });
     });
     test.it('8.4 should not add/publish api backend with invalid host name', function() {
         newUser.added = false;
         CommonUtils.signIn(driver);
-        CommonUtils.fillSignInForm(driver, 'test@test.test', 'testuser');
+        CommonUtils.fillSignInForm(driver, 'robot@test.com', 'robottest');
         CommonUtils.goToDashboard(driver);
         AddAPIBackendUtil.addNewBackend(driver, {
-            backendName: 'NewBackend',
-            description: 'Test Description',
-            hostName: 'google',
-            portNumber: '80',
-            frontendPrefix: '/test',
-            backendPrefix: '/test'
+            apiName: 'NewBackend',
+            apiDescription: 'Test Description',
+            apiURL: 'google.com'
+        });
+        // Get the URL field in form
+        var errorElement = driver.findElement(By.xpath('//*[@id="addApiForm"]/fieldset/div[3]/span'));
+        errorElement.getText().then(function(text){
+            assert.include(text, 'URL must be a valid URL');
         });
-        var errorElement = driver.findElement(By.xpath('//*[@class="s-alert-box-inner"]/p'));
+    });
+    test.it('8.5 should not add/publish api with no unique name', function() {
+        newUser.added = false;
+        CommonUtils.signIn(driver);
+        CommonUtils.fillSignInForm(driver, 'robot@test.com', 'robottest');
+        CommonUtils.goToDashboard(driver);
+        var values = {
+            apiName: 'NewBackend',
+            apiDescription: 'Test Description',
+            apiURL: 'http://google.com'
+        };
+        AddAPIBackendUtil.addNewBackend(driver, values);
+        // Get the API Name field in form
```

Comment:
Tried line driver.sleep(1000); after adding.
and test passes better 5/5 runs for nightly.